### PR TITLE
Support "-style" option

### PR DESCRIPTION
**Pull request by fwany21**
Support "-style" option

### CODE REVIEW

**Comment by jklmnn** [comment]
Why are you using `-style` instead of `-s` and `--style` like all other switches?

**Comment by fwany21** [comment]
> Why are you using `-style` instead of `-s` and `--style` like all other switches?

Because clang-format also uses '-style' to configure the style.
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

File: run-clang-format.py [hunk 1]
```diff
@@ -298,6 +301,9 @@ def main():
         default=[],
         help='exclude paths matching the given glob-like pattern(s)'
         ' from recursive search')
+    parser.add_argument(
+        '-style',
+        help='formatting style to apply (LLVM, Google, Chromium, Mozilla, WebKit)')
```
Comment (AVSurfer123) [inline run-clang-format.py:306]: Also style can be `file` to use the .clang-format file

**Comment by AVSurfer123** [comment]
The new option should match the Python option standard though, and what the other options already are.

File: run-clang-format.py (hunk 1 above)
Comment (fwany21) [reply run-clang-format.py:306]: It works when entering the format file. Please tell me if you have any problems.

**Comment by celonymire** [comment]
Can you extend this to support the "file" option for style? I can start using this again if it added that support :D .

**Comment by fwany21** [comment]
> Can you extend this to support the "file" option for style? I can start using this again if it added that support :D .

If you mean that the option "-style=file", already supports it.
However, forms like "-style=clang-format.file" are not the scope of this project.
In this case, I have to modify it directly in clang-format.

Please refer to https://stackoverflow.com/questions/46373858/how-do-i-specify-a-clang-format-file

**Comment by celonymire** [comment]
> > Can you extend this to support the "file" option for style? I can start using this again if it added that support :D .
> 
> If you mean that the option "-style=file", already supports it.
> However, forms like "-style=clang-format.file" are not the scope of this project.
> In this case, I have to modify it directly in clang-format.
> 
> Please refer to https://stackoverflow.com/questions/46373858/how-do-i-specify-a-clang-format-file

Yes I did mean "-style=file", thank you very much!

**Comment by Sarcasm** [comment]
Would you mind renaming the option `--style` (double dashes) like other options?
I'm not sure why, but on my system, clang-format actually documents the double dashes options as well, and I suspect they work on other systems too.

```
$ which clang-format
/usr/bin/clang-format
$ clang-format --version
clang-format version 10.0.1 
$ echo 'void f() {}' | clang-format --style=Mozilla
void
f()
{}
$ clang-format --help |& grep -- --style=
  --style=<string>           - Coding style, currently supports:
$ |
```

**Comment by fwany21** [comment]
> Would you mind renaming the option `--style` (double dashes) like other options?
> I'm not sure why, but on my system, clang-format actually documents the double dashes options as well, and I suspect they work on other systems too.
> 
> ```
> $ which clang-format
> /usr/bin/clang-format
> $ clang-format --version
> clang-format version 10.0.1 
> $ echo 'void f() {}' | clang-format --style=Mozilla
> void
> f()
> {}
> $ clang-format --help |& grep -- --style=
>   --style=<string>           - Coding style, currently supports:
> $ |
> ```

I've processed it. :smiley:

**Comment by Sarcasm** [comment]
Could you rebase on top of master?
I could do it myself, but I'm afraid to lose your authorship.

**Comment by fwany21** [comment]
> Could you rebase on top of master?
> I could do it myself, but I'm afraid to lose your authorship.

Oh! I did it!

**Comment by Sarcasm** [comment]
Thank you!

**Pull request merged**